### PR TITLE
fix: Add docs for passThroughArgs

### PR DESCRIPTION
**Pull request by svotaw**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

Add some docs to explain how LightGBM passThroughArgs works

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/b00e953c1306a101b763e10141facf6059b2436e/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.0/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/b00e953c1306a101b763e10141facf6059b2436e/website%2Fversioned_docs%2Fversion-0.10.0%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.1/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/b00e953c1306a101b763e10141facf6059b2436e/website%2Fversioned_docs%2Fversion-0.10.1%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.2/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/b00e953c1306a101b763e10141facf6059b2436e/website%2Fversioned_docs%2Fversion-0.10.2%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/8f3749f305e318fd36db799b3e1565301043cc32/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.0/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/8f3749f305e318fd36db799b3e1565301043cc32/website%2Fversioned_docs%2Fversion-0.10.0%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.1/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/8f3749f305e318fd36db799b3e1565301043cc32/website%2Fversioned_docs%2Fversion-0.10.1%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.2/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/8f3749f305e318fd36db799b3e1565301043cc32/website%2Fversioned_docs%2Fversion-0.10.2%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: website/versioned_docs/version-0.10.1/features/lightgbm/about.md [hunk 1]
```diff
@@ -57,6 +57,48 @@ model = LightGBMRegressor(application='quantile',
...
+```
+
+LightGBM has far more parameters than SynapseML exposes. For cases where you
+need to set some parameters that SyanpseML does not expose a setter for, use
+passThroughArgs. This is just a free string that you can use to add extra parameters
+to the command SynapseML sends to configure LightGBM.
+
+```python
+from synapse.ml.lightgbm import LightGBMClassifier
+model = LightGBMClassifier(passThroughArgs="force_row_wise=true",
```
Comment (JessicaXYWang) [inline website/versioned_docs/version-0.10.1/features/lightgbm/about.md:79]: maybe provide example of passing more than one args in python as well? Similar to how it is done with Scala example. Passing argument with whitespace as a Seperator might not be intuitive. 

model = LightGBMClassifier(passThroughArgs="force_row_wise=true min_sum_hessian_in_leaf=2e-3",

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/cd1d40518f3fd70a20cc2fc30dfef0256ae8573d/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.0/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/cd1d40518f3fd70a20cc2fc30dfef0256ae8573d/website%2Fversioned_docs%2Fversion-0.10.0%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.1/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/cd1d40518f3fd70a20cc2fc30dfef0256ae8573d/website%2Fversioned_docs%2Fversion-0.10.1%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/versioned_docs/version-0.10.2/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/cd1d40518f3fd70a20cc2fc30dfef0256ae8573d/website%2Fversioned_docs%2Fversion-0.10.2%2Ffeatures%2Flightgbm%2Fabout.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by mhamilton723 — APPROVED** [review]

File: website/versioned_docs/version-0.10.1/features/lightgbm/about.md (hunk 1 above)
Comment (mhamilton723) [reply website/versioned_docs/version-0.10.1/features/lightgbm/about.md]: Great idea @JessicaXYWang !

**Pull request merged**